ngraph/core/include/ngraph/op/proposal.hpp

```diff
@@ -46,7 +46,7 @@ namespace ngraph
             size_t min_size = 1;
             std::vector<float> ratio;
             std::vector<float> scale;
-            bool clip_before_nms = false;
+            bool clip_before_nms = true;
```

Comment:
@lazarevevgeny is it ok that we change default value for opset1 op ?

Reply:
Previous version was not right. Now it's ok according to docs https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/detection/Proposal_4.md

Reply:
The nGraph implementation was wrong. Both documentation and CPU/GPU implementation have the default value equal to `True`. So I consider this as a bug which must be solved. Therefore I agree with this change.